localization/english/triggers/wc_character_triggers_l_english.yml

```diff
@@ -36,6 +36,13 @@
  has_trait_with_flag_can_not_romance_first:0 "You are #bold not#! allowed to romance"
  has_trait_with_flag_can_not_romance_third:0 "[CHARACTER.GetShortUIName|V] is #bold not#! allowed to romance"
 
+ has_trait_with_flag_no_aging_penalties_global:0 "Is #bold not#! aging"
```

Comment:
I meant `TRAIT_FLAG_DESC_<flag name>` localization, but this is nice too. Keep it.